packages/console/src/browser/console-widget.ts

```diff
@@ -139,6 +149,12 @@ export class ConsoleWidget extends BaseWidget implements StatefulWidget {
         input.getControl().createContextKey('consoleInputFocus', true);
         const contentContext = this.contextKeyService.createScoped(this.content.node);
         contentContext.setContext('consoleContentFocus', true);
+
+        this.toDispose.pushAll([
+            this._modelChangeListener,
```

Comment:
Suggestion: It's not necessary to add `this._modelChangeListener` here, see above.

Reply:
Fixed with https://github.com/eclipse-theia/theia/pull/14854/commits/104fe67f91fee6baa24536920fd7a12689f156bf.